File: SafeCityAPI/src/main/java/com/ph/SafeCityApiApplication.java
package com.ph;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@SpringBootApplication
@EnableAsync
public class SafeCityApiApplication {

	public static void main(String[] args) {
		SpringApplication.run(SafeCityApiApplication.class, args);
	}
}

File: SafeCityAPI/src/main/kotlin/com/ph/SafeCityApiApplication.kt
package com.ph

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableAsync

@EnableAsync
@SpringBootApplication
open class SafeCityApiApplication
fun main(args: Array<String>) {
    SpringApplication.run(SafeCityApiApplication::class.java, *args)
}

